src/base/bittorrent/tracker.cpp

```diff
@@ -271,52 +275,95 @@ void Tracker::processAnnounceRequest()
     // ip address
     announceReq.socketAddress = m_env.clientAddress;
     announceReq.claimedAddress = queryParams.value(ANNOUNCE_REQUEST_IP);
+    // request port
+    announceReq.requestPort =  m_env.clientPort;
 
     // Enforce using IPv4 if address is indeed IPv4 or if it is an IPv4-mapped IPv6 address
     bool ok = false;
     const qint32 decimalIPv4 = announceReq.socketAddress.toIPv4Address(&ok);
-    if (ok)
+    if (ok) {
         announceReq.socketAddress = QHostAddress(decimalIPv4);
+        LogMsg(tr(R"(Embedded Tracker: Announce request from non- plain-IPv6 address %1.)"
```

Comment:
I don't see why you need to use raw string literal.

Reply:
Using raw string literals prevents escape hell. It's more consistent to use it everywhere, if we already use it once. What are the disadvantages?

Reply:
Downvotes without explanation are not helpful.

Reply:
>It's more consistent to use it everywhere, if we already use it once.

This is a very destructive idea, IMO. Everything must have its place and time. It is not designed as a replacement for regular string literals.

Reply:
>Downvotes without explanation are not helpful.

Then they would be useless...

Reply:
@glassez 
> > Downvotes without explanation are not helpful.
> 
> Then they would be useless...

Only if we start acting like redditors, which I hope doesn't happen.

> This is a very destructive idea, IMO. Everything must have its place and time. It is not designed as a replacement for regular string literals.

But for these specific usages, what is the problem? By everywhere, I meant "everywhere in the embedded tracker code", changing it everywhere in the code base might indeed not be desirable, but that's out of scope of this PR anyway.

I can understand undoing this change here if you are of the opinion that "changing to raw string literals where it makes sense should be done at once for the whole code base in another PR". Is that the case?

Reply:
> By everywhere, I meant "everywhere in the embedded tracker code"

It doesn't change my answer.
>I can understand undoing this change here if you are of the opinion that "changing to raw string literals where it makes sense should be done at once for the whole code base in another PR". Is that the case?

No.
I just mean that use if raw string literal has no sense in case of simple string like this one. And some kind of unification is not appropriate here.

Reply:
@glassez Ok, I'll change it to normal literals where it is not needed.

Reply:
Ok, now raw string literals are only used when the strings contain `"`. Do you think I should revert to using escaped strings also in that case?

Reply:
@glassez ping, like I said, raw string literals are now only used when justifiable (i.e. when the string itself also contains `"`). Is this acceptable, or do you really want me to revert to using escaped strings all around?